config/samples/resources/tagslocationtagbinding/tags_v1beta1_tagslocationtagbinding.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: tags.cnrm.cloud.google.com/v1beta1
+kind: TagsLocationTagBinding
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: ${PROJECT_ID?}
+  name: location-tag-binding-sample
+spec:
+  parentRef:
+    name: ${PROJECT_ID?}-tag-dep
```

Comment:
I do not think I understand this. ${PROJECT_ID?}-tag-dep is a StorageBucket resource(defined in above yaml), why we use it as parent of resource TagsLocationTagBinding?

Reply:
`parent` for the binding resource is the target, where the binding will attach the TagValue. Any [tag-compatible](https://cloud.google.com/resource-manager/docs/tags/tags-supported-services) resource can be defined as a parent. Instead of using `ref`, I can define the binding target through the `external` parameter, but I thought defining the whole chain through KCC resources would be more useful.

Reply:
Discussed in another comment, mark this as resolved.